dictionary_type : 4
marker_edge_size : 0.04
marker_ids : [ 0 ]

# dictionary types are as follows:
# cv::aruco::DICT_4X4_50   = 0
# cv::aruco::DICT_4X4_100  = 1
# cv::aruco::DICT_4X4_250  = 2
# cv::aruco::DICT_4X4_1000 = 3
# cv::aruco::DICT_5X5_50   = 4
# cv::aruco::DICT_5X5_100  = 5
# cv::aruco::DICT_5X5_250  = 6
# cv::aruco::DICT_5X5_1000 = 7
# cv::aruco::DICT_6X6_50   = 8
# cv::aruco::DICT_6X6_100  = 9
# cv::aruco::DICT_6X6_250  = 10
# cv::aruco::DICT_6X6_1000 = 11
# cv::aruco::DICT_7X7_50   = 12
# cv::aruco::DICT_7X7_100  = 13
# cv::aruco::DICT_7X7_250  = 14
# cv::aruco::DICT_7X7_1000 = 15